.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: '00:00'
+  open-pull-requests-limit: 10
+  reviewers:
+    - adamwathan
+  assignees:
+    - adamwathan
+  commit-message:
+    prefix: fix
+    prefix-development: chore
+    include: scope
+- package-ecosystem: github-actions
```

Comment:
I would recommend putting a newline before the second `package-ecosystem` (`github -actions`), so that the 2 blocks are clearly separated. That will make it easier to see which configuration belongs to which `package-ecosystem`.